src/containers/paper-canvas.jsx

```diff
@@ -264,7 +266,10 @@ class PaperCanvas extends React.Component {
     render () {
```

Comment:
Rather than adding ModeClasses to modes.js, can you add a switch statement on this.props.mode here, to choose the css class string?

modes.js is in lib/ which implies that it should be stand-alone, so it's odd for it to refer to specific CSS in scratch-paint.